x/checkers/keeper/msg_server_play_move.go

```diff
@@ -2,16 +2,61 @@ package keeper
 
 import (
 	"context"
+	"errors"
+	"strings"
 
 	sdk "github.com/cosmos/cosmos-sdk/types"
+	rules "github.com/xavierlepretre/checkers/x/checkers/rules"
 	"github.com/xavierlepretre/checkers/x/checkers/types"
 )
 
 func (k msgServer) PlayMove(goCtx context.Context, msg *types.MsgPlayMove) (*types.MsgPlayMoveResponse, error) {
 	ctx := sdk.UnwrapSDKContext(goCtx)
 
-	// TODO: Handling the message
-	_ = ctx
+	storedGame, found := k.Keeper.GetStoredGame(ctx, msg.IdValue)
+	if !found {
+		return nil, errors.New("Game not found " + msg.IdValue)
```

Comment:
sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"

could sdkerrors be used? 

could be something like 

```
sdkerrors.Wrapf(sdkerrors.ErrUnauthorized, "game not found %s", msg.IdValue)```

Reply:
Among the `sdkerrors`, this `ErrUnauthorized` looks like the only one that I could use for anything related to checkers, no? Is it recommended to use these `sdkerrors`?

Reply:
yes, sdkerrors are recommended. Modules are meant to define their own type of errors as well.

Reply:
Nice, I see the `x/checkers/types/errors.go`. Is there a convention with the numbering?